app/models.py

```diff
@@ -83,13 +84,23 @@ class PlanEnum(enum.Enum):
     yearly = 3
 
 
+class AliasGeneratorEnum(enum.Enum):
+    word = 1  # aliases are generated based on random words
+    uuid = 2  # aliases are generated based on uuid
+
+    @classmethod
+    def has_value(cls, value: int) -> bool:
+        return value in set(item.value for item in cls)
+
+
 class User(db.Model, ModelMixin, UserMixin):
     __tablename__ = "users"
     email = db.Column(db.String(128), unique=True, nullable=False)
     salt = db.Column(db.String(128), nullable=False)
     password = db.Column(db.String(128), nullable=False)
     name = db.Column(db.String(128), nullable=False)
     is_admin = db.Column(db.Boolean, nullable=False, default=False)
+    alias_generator = db.Column(db.Integer, nullable=True, default=AliasGeneratorEnum.word.value)
```

Comment:
Maybe we can use `nullable=False` here? A field is easier to work with when it's non-nullable, that's why fields are generally non-nullable. Exception made for the ones that null value makes sense like `profile_picture_id`, `custom_domain_id`.